src/Node/Render.php

```diff
@@ -62,12 +62,19 @@ public static function getEnvironment(): object {
    */
   protected function addGetTemplate(Twig_Compiler $compiler) {
     $this->setEnvironment($compiler->getEnvironment());
-    $compiler->raw('$component = new \Drupal\twig_fractal\Component(');
-    $compiler->raw('\Drupal\twig_fractal\Node\Render::getEnvironment(),');
-    $compiler->subcompile($this->getNode('expr'));
-    $compiler->raw(')')->raw(";\n");
-    $compiler->raw('$defaults = $component->getDefaultVariables();');
-    $compiler->raw('$passed_variables = [];');
+    $compiler->raw('$handles = (array) ')->subcompile($this->getNode('expr'))->raw(';');
+    $compiler->raw('foreach($handles as $handle):');
+      $compiler->raw('$passed_variables = $defaults = [];');
+      $compiler->raw('$component = new \Drupal\twig_fractal\Component(');
+      $compiler->raw('\Drupal\twig_fractal\Node\Render::getEnvironment(),');
```

Comment:
The indentation is not consistent; you started to indent for the new `foreach` but didn't continue with the arguments here.